src/main/java/domain/Height.java

```diff
@@ -0,0 +1,19 @@
+package domain;
+
+import constant.Exception;
+
+public class Height {
+
+    public static final int MIN_OF_HEIGHT = 1;
+    public static final int MAX_OF_HEIGHT = 100;
+
+    public Height(int height) {
+        validate(height);
+    }
+
+    private void validate(int height) {
+        if (height < MIN_OF_HEIGHT || height > MAX_OF_HEIGHT) {
```

Comment:
가독성 슈퍼 미세팁인데요, 꺽쇠 방향을 일치시켜주면 조금 더 잘 읽힙니다
```suggestion
        if (height < MIN_OF_HEIGHT || MAX_OF_HEIGHT < height) {
```